Config.psm1

```diff
@@ -21,57 +21,74 @@ Import-Module "$localResourcesDir\ini.psm1"
 function Get-availableConfigOptionOptions {
     return @(
         @{"Name" = "wim_file_path"; "DefaultValue" = "D:\Sources\install.wim";
-          "Description" = "Wim file path."},
+          "Description" = "The location of the WIM file from the Windows ISO."},
         @{"Name" = "image_name"; "DefaultValue" = "Windows Server 2012 R2 SERVERSTANDARD";
-          "Description" = "Wim image name"},
+          "Description" = "This is the complete name of the Windows version that will be generated. 
+                           In order to find the possible options, use the Get-WimFileImagesInfo command 
+                           and look for the Name property."},
         @{"Name" = "image_path"; "DefaultValue" = "${ENV:TEMP}\win-image.vhdx";
-          "Description" = "Image path"},
+          "Description" = "The destination of the generated image."},
         @{"Name" = "virtual_disk_format"; "DefaultValue" = "VHDX";
-          "Description" = "Virtual disk format"},
+          "Description" = "Select between VHD, VHDX, QCOW2, VMDK or RAW formats."},
         @{"Name" = "image_type"; "DefaultValue" = "HYPER-V";
-          "Description" = ""},
+          "Description" = "This parameter allows to choose between MAAS, KVM and Hyper-V specific images."},
```

Comment:
For HYPER-V, cloudbase-init will be installed and the generated image should be in vhd or vhdx format.
For MAAS, in addition to cloudbase-init, the curtin tools are installed and the generated image should be in raw.tgz format.
For KVM,  in addition to cloudbase-init, the VirtIO drivers are installed and the generated image should be in qcow2 format.